lucille-examples/lucille-s3-ingest-example/conf/s3-opensearch.conf

```diff
@@ -0,0 +1,73 @@
+connectors: [
+ {
+    name: "fileConnector",
+    class: "com.kmwllc.lucille.connector.FileConnector",
+    pipeline: "pipeline1",
+    pathToStorage: ${PATH_TO_STORAGE}
+    cloudOptions: {
+      region: ${AWS_REGION}
+      accessKeyId: ${AWS_ACCESS_KEY_ID}
+      secretAccessKey: ${AWS_SECRET_ACCESS_KEY}
+    }     
+ }
+]
+
+pipelines: [
+  {
+    name: "pipeline1",
+    stages: [
+     {
+       name:"TextExtractor"
+       class: "com.kmwllc.lucille.tika.stage.TextExtractor"
+       byte_array_field: "file_content"
```

Comment:
Might be worth commenting that FileConnector places the content in this "file_content" field.